shared/Microsoft.AspNetCore.RangeHelper.Sources/RangeHelper.cs

```diff
@@ -30,7 +32,8 @@ internal static class RangeHelper
         public static (bool, RangeItemHeaderValue) ParseRange(
             HttpContext context,
             RequestHeaders requestHeaders,
-            long length)
+            long length,
+            ILogger logger)
```

Comment:
In terms of not breaking the build - consider checking in an overload with a null logger and removing it once the package is on the feed This should avoid local builds from being broken.

Reply:
Sound great 👍